features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/modules/statistics/usage.jag

```diff
@@ -86,18 +102,27 @@ function getProviderAPIUsage(providerName,from,to, server) {
 function getProviderAPIUserUsage(providerName, apiName, server) {
     var usage = [];
     var usageout = [];
+
+    if(usageClient == null){
+        return {
+            error:false,
+            usage:[]
+        };
+    }
+
     try {
-        usage = apiProvider.getProviderAPIUserUsage(providerName, apiName, server);
+        usage = usageClient.getUsageBySubscribers(providerName, apiName, 10);
         if (log.isDebugEnabled()) {
             log.debug("getProviderAPIUserUsage for : " + server);
         }
         if (usage == null) {
             return {
-                error:true
+                error:false,
+                usage:[]
             };
 
         } else {
-
+            usage=JSON.parse(gson.toJson(usage));
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea